pywren/storage/storage.py

```diff
@@ -67,6 +68,17 @@ def put_func(self, key, func):
         """
         return self.backend_handler.put_object(key, func)
 
+    def put_cancelled(self, callset_id, call_id, cancelled_data):
+        """
+        Put data to signal cancel into storage
+        :param key: cancel_key
+        :param func: cancelled_data
+        :return: None
+        """
+        key = create_cancel_key(self.prefix, callset_id,
```

Comment:
why do we call create_cancel_key again here ? isn't create_keys called before put_cancelled is ?

Reply:
I think you are right about this. Will remove in the next update.

Reply:
Actually, I think the reason behind this is that the cancel key isn't accessible from a `future`, whereas the `call_id` and `callset_id` are. Since `create_cancel_key` is just a path join of these ids. I will leave this in here.